src/inputs/ObjectSearchCreate.tsx

```diff
@@ -115,6 +110,24 @@ class ObjectSearchCreate extends Component<IObjectSearchCreateProps> {
       </Antd.Form.Item>
     );
   }
+
+  public render () {
+    const className = cx(
+      CX_PREFIX_SEARCH_CREATE,
+      {[`${CX_PREFIX_SEARCH_CREATE}-create`]: this.isAddingNew.isTrue },
+      this.props.className,
+    );
+
+    return (
+      <div className={className}>
```

Comment:
Now properly setting `className` for both search and add modes, on the same level, same component.